masterlist.yaml

```diff
@@ -18945,17 +18948,22 @@ plugins:
       - link: 'https://www.nexusmods.com/skyrim/mods/22016/'
         name: 'Vampire Kinship'
     dirty:
-      - <<: *dirtyPlugin
-        crc: 0xc1417f3b
+      - <<: *quickClean
+        crc: 0xC1417F3B
+        util: '[TES5Edit v4.0.4b](https://www.nexusmods.com/skyrim/mods/25859)'
         itm: 4
-  - name: 'Vampire''s Underwater suffocation FIX 1.1.esp'
+  - name: 'Vampire''s Underwater suffocation FIX 1.2( NoDawnguard)?\.esp'
```

Comment:
The first `.` should be escaped too like `Vampire''s Underwater suffocation FIX 1\.2( NoDawnguard)?\.esp`.

Reply:
Thanks, I'll adjust it with the next PR.